monkey/infection_monkey/post_breach/job_scheduling/linux/job_scheduling.py

```diff
@@ -0,0 +1,11 @@
+TEMP_CRON = "$HOME/monkey-schedule-jobs"
+
+
+def get_linux_commands_to_schedule_jobs():
+    return [
+        'touch {} &&'.format(TEMP_CRON),
+        'crontab -l > {} &&'.format(TEMP_CRON),
+        'echo \"# Successfully scheduled a job using crontab\" |',
+        'tee -a {} &&'.format(TEMP_CRON),
+        'crontab {}'.format(TEMP_CRON)
```

Comment:
1. Use python3 formatting `f"touch {TEMP_CRON} &&"`
2. you came up with this yourself? what tests did you do? Which command deletes `$HOME/monkey-schedule-jobs` file?

Reply:
Yes, came up with it myself; updated - `$HOME/monkey-schedule-jobs` is deleted now.